arc/job/adapters/obabel.py

```
@@ -0,0 +1,270 @@
+            if "econv" in settings["opt_gradient_settings"].keys() and not isinstance(settings["opt_gradient_settings"]["econv"], (float, int)):
+                return False
+        return True
+
+    def set_files(self) -> None:
+        """
+        Set files to be uploaded and downloaded. Writes the files if needed.
+        Modifies the self.files_to_upload and self.files_to_download attributes.
+
+        self.files_to_download is a list of remote paths.
+
+        self.files_to_upload is a list of dictionaries, each with the following keys:
+        ``'name'``, ``'source'``, ``'make_x'``, ``'local'``, and ``'remote'``.
+        If ``'source'`` = ``'path'``, then the value in ``'local'`` is treated as a file path.
+        Else if ``'source'`` = ``'input_files'``, then the value in ``'local'`` will be taken
+        from the respective entry in inputs.py
+        If ``'make_x'`` is ``True``, the file will be made executable.
+        """
+        pass
+
+    def set_additional_file_paths(self) -> None:
+        """
+        Set additional file paths specific for the adapter.
+        Called from set_file_paths() and extends it.
+        """
+        self.local_path_to_output_file = os.path.join(self.local_path, 'output.yml')
+
+    def set_input_file_memory(self) -> None:
+        """
+        Set the input_file_memory attribute.
+        """
+        pass
+
+    def execute_incore(self):
+        """
+        Execute a job incore.
+        """
+        self._log_job_execution()
+
+        if "freq" in self.job_type:
+            raise NotImplementedError("Openbabel does not contain a frequency calculation option.\n"
+                                      "Use another ESS from the available options.")
+        
+        if self.level.method is not None:
+            ob_default_settigns["FF"] = self.level.method
+
+        if not self.check_settings(ob_default_settigns):
+            logger.error(f'Given wrong settings in input: {ob_default_settigns}\n'
```

I think we should raise an error here, not just log an error